src/server_manager/electron_app/digitalocean_oauth.ts

```diff
@@ -147,18 +146,18 @@ export function runOauth(): OauthSession {
               <script>
                   // We can't use URLSearchParams in IE :-(
                   function splitParams(paramsStr) {
-                    var params = {};
-                    var kvs = paramsStr.split("&");
-                    for (var i in kvs) {
+                    const params = {};
```

Comment:
This code has to run in arbitrary old web browsers so it has to use `var`.

Reply:
+1 Please revert this file.